polaris-react/src/components/AlphaHoverCard/AlphaHoverCard.stories.tsx

```diff
@@ -0,0 +1,679 @@
+                        >
+                          <Box
+                            borderColor="border-brand"
+                            borderRadius="500"
+                            background="bg-fill-secondary"
+                            paddingInline="150"
+                          >
+                            <Text
+                              as="span"
+                              variant="bodyXs"
+                            >{`${quantity}`}</Text>
+                          </Box>
+                        </Box>
+                      </div>
+                    </div>
+                    <BlockStack gap="100">
+                      <Link url="#" monochrome removeUnderline>
+                        <Text as="p" variant="bodySm">
+                          {title}
+                        </Text>
+                      </Link>
+                      <Text as="p" tone="subdued" variant="bodySm">
+                        {variant}
+                      </Text>
+                    </BlockStack>
+                  </InlineStack>
+                </Box>
+              ),
+            )}
+          </Box>
+        </BlockStack>
+      </Box>
+    );
+  };
+
+  const rowMarkup = [...orders].map(
+    (
+      {
+        id,
+        title,
+        date,
+        customer,
+        channel,
+        total,
+        paymentStatus,
+        fulfillmentStatus,
+        items,
+        deliveryStatus,
+        deliveryMethod,
+        location,
+        tags,
+      },
+      index,
+    ) => {
+      return (
+        <IndexTable.Row
+          id={id}
+          key={id}
+          selected={selectedResources.includes(id)}
+          position={index}
+        >
+          <IndexTable.Cell>{title}</IndexTable.Cell>
+          <IndexTable.Cell>{date}</IndexTable.Cell>
+          <IndexTable.Cell
+            flush
+            preview={renderCustomerCellPreview(customer, id)}
+          >
+            <div style={{minHeight: '100%', padding: 'var(--p-space-150)'}}>
+              <Link monochrome removeUnderline url="#">
+                {customer.name}
+              </Link>
+            </div>
+          </IndexTable.Cell>
+          <IndexTable.Cell>{channel}</IndexTable.Cell>
+          <IndexTable.Cell>
+            <Text as="span" alignment="end" numeric>
+              {total}
+            </Text>
+          </IndexTable.Cell>
+          <IndexTable.Cell>{paymentStatus}</IndexTable.Cell>
+          <IndexTable.Cell>{fulfillmentStatus}</IndexTable.Cell>
+          <IndexTable.Cell
+            flush
+            preview={renderItemsCellPreview({
+              id,
+              location,
+              deliveryMethod,
+              fulfillmentStatus,
+              items,
+            })}
+          >
+            <div style={{minHeight: '100%', padding: 'var(--p-space-150)'}}>
+              <Link monochrome removeUnderline url="#">
+                {`${items.length} items`}
+              </Link>
+            </div>
```

Comment:
This was a copy/paste typo from copying the customers activator, this shouldn't be a link. cc @heyjoethomas 

```suggestion
                {`${items.length} items`}
            </div>
```